documentation/AuthorizationService.md

```diff
@@ -1,14 +1,15 @@
 ## Authorization Service
 
-The authorization service provides API that allows users to add, retrieve, update the data used by the [GPII OAuth2 security server](https://wiki.gpii.net/w/GPII_OAuth_2_Guide). These data include OAuth clients, users, users' authorization decisions and access tokens used by client credentials grant.
+The authorization service provides API that allows users to add, retrieve, update the data used by the [GPII OAuth2 authorization server](https://wiki.gpii.net/w/GPII_OAuth_2_Guide). These data include OAuth clients, users, authorizations used by various OAuth2 grants.
 
-###APIs for [Authorization Code Grant](https://wiki.gpii.net/w/GPII_OAuth_2_Guide#Authorization_Code_Grant)
+### APIs for [Authorization Code Grant](https://wiki.gpii.net/w/GPII_OAuth_2_Guide#Authorization_Code_Grant)
 
-#### addAuthorization(userId, oauth2ClientId, selectedPreferences)
-* **description**: Add the authorization decision. Check if the user authorization decision already exists based on the user id and OAuth2 client id. If it doesn't exist, generate an access token and add the authorization decision. If it does exist, do nothing. 
+#### addUserAuthorizedAuthorization(userId, solutionId, oauth2ClientId, selectedPreferences)
```

Comment:
It looks like this function is used for both Web Preferences Consumers and Onboarded Solutions?

We might need a section then for "APIs for Onboarded Solutions" that references this function, or a section for "APIs applicable to both Web Preferences Consumers and Onboarded Solutions".